src/cpp/include/openvino/genai/whisper_speech_recognition_pipeline.hpp

```diff
@@ -0,0 +1,106 @@
+// Copyright (C) 2023-2024 Intel Corporation
+// SPDX-License-Identifier: Apache-2.0
+
+#pragma once
+
+#include <openvino/openvino.hpp>
+#include <optional>
+#include <variant>
+
+#include "openvino/core/any.hpp"
+#include "openvino/genai/llm_pipeline.hpp"
+#include "openvino/genai/whisper_generation_config.hpp"
+
+namespace ov::genai {
+
+using OptionalWhisperGenerationConfig = std::optional<WhisperGenerationConfig>;
+using PCMf32AudioDataInput = std::vector<float>;
```

Comment:
should / can it be ov::Tensor instead?

Reply:
I think we agreed to use `RawSpeechInput` which is `std::vector<float>` as input. Please tell me if this comment is still valid.